tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -62,7 +113,22 @@ def do_build(args: CensusBuildArgs, skip_completed_steps: bool = False) -> int:
         do_the_release,
         do_report_copy,
         do_old_release_cleanup,
+        do_log_copy,
     ]
+    rv = _do_build(build_steps, args, skip_completed_steps)
+    logging.info("Census full build: completed")
+    return rv
+
+
+def _do_build(
```

Comment:
at this point, this function name is confusing and doesn't match what the function does.   Suggest a rename.

Maybe:   _do_steps ?   I"m agnostic, but it is confusing to see do_release call _do_build

Reply:
Absolutely agree. `_do_steps` looks great.